syntax = "proto2";

import "dccl/option_extensions.proto";

package jaiabot.sensor.protobuf;

message AtlasScientificOEMEC
{
    // REQUIRES DCCL 4.2.2 - uncomment when we wrap this in
    // [(dccl.field) = {
    // units {
    //         custom {
    //             unit: "jaiabot::units::microsiemens_per_cm_unit"
    //             header: "jaiabot/units/conductivity.h"
    //         }
    //     }
    // }];
    optional double conductivity = 1;           // Î¼S/cm
    optional double total_dissolved_solids = 2; // ppm
    optional double salinity = 3;               // PSU (ppt)
}



